app/views/comments/event_comments.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1 class="text-center my-4">Event Comment</h1>
+<table class="table w-50 table-bordered table-striped table-hover m-auto text-center">
+  <tr>
+    <th>User</th>
+    <th>Comments</th>
+    <th>Comment Action</th>
+    <th>Like</th>
+    <th>Like Count</th>
+  </tr>
+  <% @comments.each do |comment| %>
+    <tr>
+      <td><%= comment.user.username %></td>
+      <td><%= comment.user_comment %></td>
+      <% if comment.user_id == @current_user.id %>
+        <td><%= link_to 'Edit', edit_comment_path(comment.id), class: "text-decoration-none text-dark btn btn-warning me-2" %>
+        <%= link_to "Destroy", comment_path(comment.id), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" }, class: "text-decoration-none text-light btn btn-danger" %></td>
+      <% else %>
+        <td></td>
+      <% end %>
+      <% if comment.user.id != @current_user.id %>
```

Comment:
Is `comment.user.id and comment.user_id` both are different?
Also, if you want to use `comment.user.id` to check the condition then please use the `comment.user != @current_user`